### PR TITLE
docs(tooltip): tooltip needs cleanup

### PR DESCRIPTION
**Pull request by nikkimk**
## What I did

1. Updated tooltip docs.

## Testing Instructions

1. View [DP tooltip docs](https://deploy-preview-972--red-hat-design-system.netlify.app/elements/tooltip/).
2. Compare to [tooltip doc](https://docs.google.com/document/d/1HUa31FFVjv0Sl1qSt0qABqLyBsJ5hcL8ByxlT5Bg2Ko/edit#) and to [Docs cleanup checklist](https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist).

## Notes to Reviewers
The following are WIP to be addressed as other issues/PRs:
- [Related items list](https://github.com/RedHat-UX/red-hat-design-system/pull/942)
- [Image width styling](https://github.com/RedHat-UX/red-hat-design-system/pull/891)
- [Accessibility tab](https://github.com/RedHat-UX/red-hat-design-system/issues/968)
- [Code tab styling](https://github.com/RedHat-UX/red-hat-design-system/issues/969)

### CODE REVIEW

File: .changeset/nasty-flies-flash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+Added a summary to `<rh-tooltip>`
```
Comment (bennypowers) [inline .changeset/nasty-flies-flash.md:5]:
```suggestion
`<rh-tooltip>`: corrected and updated custom elements manifest
```

File: elements/rh-tooltip/rh-tooltip.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ import styles from './rh-tooltip.css';
 
 /**
  * Tooltip
- * @summary Toggles a small overlay of text only on hover or focus
+ * @summary A tooltip is a floating text area that provides helpful or contextual information on hover, focus, or tap.
  * @slot - Place element content here
```
Comment (bennypowers) [inline elements/rh-tooltip/rh-tooltip.ts:17]: can we add in the `@cssprops` here, too, or is that out of scope?

**Review by bennypowers — CHANGES_REQUESTED** [review]

**Comment by bennypowers** [comment]
apologies for adding unrelated changes here, but one thing led to another:
- the bem-style css props in tooltip were contrary to guidelines, so we need a minor release to fix them
- that introduced css linting issues which needed to be addressed 
- fixing those opened up the opportunity to apply more consistent formatting across the repo

the changes outside of tooltip should all be whitespace, so shouldn't require deep review

the deprecations in tooltip require a bit more concentration

**Comment by nikkimk** [comment]
> apologies for adding unrelated changes here, but one thing led to another:
> 
> * the bem-style css props in tooltip were contrary to guidelines, so we need a minor release to fix them
> * that introduced css linting issues which needed to be addressed
> * fixing those opened up the opportunity to apply more consistent formatting across the repo
> 
> the changes outside of tooltip should all be whitespace, so shouldn't require deep review
> 
> the deprecations in tooltip require a bit more concentration

Totally fine by me. I made the changeset changes as requested. So lmk if you think this is good to go.

**Review by bennypowers — APPROVED** [review]

**Comment by coreyvickery** [comment]
@nikkimk Not sure why this was closed.

### Overview tab
- Wrong copy under `Overview` heading
- `When to use` heading, third bullet, remove the `-`

### Style tab
- `Anatomy` heading, correct styling for numbers: `Red Hat Text, 14px, 1.5 (21) line height, #4D4D4D`
- `Anatomy` heading, left align the numbers to the image container above
- Spacer images in table are still 2x
- Duplicate cross-links

### Guidelines tab
- Duplicate cross-links

### Code tab
- Duplicate cross-links

### Accessibility tab
- Missing

**Pull request merged**